Eagle-Rock-Analytics/historical-obs-platform
# Updates and renames primary stationlist for figures/references

This PR updates the 3 existing figure notebooks to rename the combined stationlists for each stage from "master" to "all". Annotations were updated for the pull stage, and a pathway to the qaqc_training csv 

--
**To test**
Run through the three notebooks and ensure that the stationlists update on AWS, as well as producing the necessary figures.
